captum/attr/_utils/attribution.py

```diff
@@ -111,6 +111,22 @@ def compute_convergence_delta(self, attributions, *args):
             "Deriving sub-class should implement" " compute_convergence_delta method"
         )
 
+    @classmethod
+    def get_name(cls):
+        r"""
+        Create readable class name by inserting a space before any capital
+        characters besides the very first.
+
+        Returns:
+            str: a readable class name
+        """
```

Comment:
nit: perhaps you can provide an example of the readable name here ?

Reply:
Updated